lib/auth/grpcserver.go

```diff
@@ -1135,6 +1135,62 @@ func (g *GRPCServer) DeleteAllDatabaseServers(ctx context.Context, req *proto.De
 	return &emptypb.Empty{}, nil
 }
 
+// GetAllDatabaseServices returns all registered DatabaseServices.
+func (g *GRPCServer) GetAllDatabaseServices(ctx context.Context, _ *emptypb.Empty) (*types.DatabaseServiceV1List, error) {
```

Comment:
Could add simple tests for the new methods too? Just checking that we get unimplemented errors back is enough - this is more so when we implement them, we already have the tests to fill in.

Reply:
I think we would require an authenticated gRPC client in order to call those methods.
Calling `GRPCServer.XxxDatabaseService` directly fails because of the `GRPCServer.authenticate` method

Can we leave this tests to the next PR (which is already in the making and has those tests)?

Reply:
Let's punt it to the next one. Thanks for looking into it.